eggo/dag.py

```diff
@@ -331,15 +384,33 @@ def output(self):
 class BAM2ADAMTask(Task):
 
     def requires(self):
+        conf = ToastConfig().config
+        parallelism = conf['numPartitionsHint'] if 'numPartitionsHint' in conf else 1
         basic = ADAMBasicTask(adam_command='transform',
                               allowed_file_formats=['sam', 'bam'])
         flat = ADAMFlattenTask(adam_command='transform',
                                allowed_file_formats=['sam', 'bam'])
+        locuspart = ADAMPartitionTask(adam_command='transform',
+                                      allowed_file_formats=['sam', 'bam'],
+                                      source_edition='basic',
```

Comment:
fine for now, but perhaps there's a way to do this with inheritance/mixins instead.